Sources/PublishCLICore/WebsiteRunner.swift

```diff
@@ -55,6 +65,13 @@ private extension WebsiteRunner {
         catch { throw CLIError.outputFolderNotFound }
     }
 
+    /// The major version number of the current system's Python install (whatever Python responds to the `python` command)
+    var systemPythonMajorVersionNumber: Int? {
+        let pythonVersionString = try? shellOut(to: "python --version")
+        let majorVersionNumberCharacter = pythonVersionString?.dropFirst(7).first
```

Comment:
I don't think it's a good idea to assume that the Python version string will always have this exact format. If they add a single whitespace or other character, this code will break. How about splitting the string by `" "` instead, and using the last component of that split to compute the version number?

Reply:
Cool yeah, I didn't really like how I was doing this either, but couldn't think of a better option. Your idea to split it makes good sense. It still could break, but is probably less likely to break.